src/main/java/com/stuypulse/robot/subsystems/ColorSensor.java

```diff
@@ -91,6 +91,21 @@ public CurrentBall getUpdateFromDriverStation() {
                             "DriverStation.getAlliance() returned invalid!", true);
                 }
 
+                return getAllianceColorFromSmartDashboard();
+        }
+    }
+
+    public CurrentBall getAllianceColorFromSmartDashboard() {
```

Comment:
you might want to store driverstation's allaicne color on the network, so that you can just swap the call to the network chooser for DriverStation.getAlliance() at the beginning of getUpdateFromDriverStation (although the name would be slightly misleading) 

this is just an implementation suggestion so it's ultimately up to whatever u decide is nicest